styles.css

```diff
@@ -169,28 +171,38 @@ body {
 
 footer {
   color: var(--color-isabelline);
-  height: 40vh;
+  height:  100%;
```

Comment:
I think you could leave vh --> 100vh since all section height is set to vh

Reply:
for Contact us height 100vh is way too much, there is not so much content

Reply:
Leave it like this